provider/pkg/provider/invoke_kustomize.go

```diff
@@ -51,10 +54,19 @@ func kustomizeDirectory(directory string, clientSet *clients.DynamicClientSet) (
 	opts := krusty.MakeDefaultOptions()
 	opts.DoLegacyResourceSort = true
 
+	// Add support for helmCharts plugin
```

Comment:
Should we wrap this behind an env var, perhaps? or is the error check on 67 enough?

Reply:
I tested it locally, and it wasn't trying to call `helm` unless the `helmCharts` feature is used in the kustomization. I think the error check should be sufficient.

Reply:
@rquitales and I discussed this offline and decided to make helmChart support experimental behind a feature flag to avoid possible impact to other users who don't need this feature. I updated https://github.com/pulumi/pulumi-kubernetes/issues/2470 with additional steps required to move this to a GA feature.